controllers/suite_test.go

```diff
@@ -71,6 +72,7 @@ var _ = BeforeSuite(func() {
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths:     []string{filepath.Join("..", "config", "crd", "bases")},
 		ErrorIfCRDPathMissing: true,
+		UseExistingCluster:    ptr.To(true),
```

Comment:
@alexsnaps no more
```go
UseExistingCluster:    &[]bool{true}[0]
```
thanks to `k8s.io/utils/ptr`'s `To` method

Reply:
we talked about that, in the end a copy is needed. constants cannot be referenced by address

Reply:
Right, I get it and yet… very sad this all leaves me… 
It would have been funnier and sad at the same time to have some compiler hint to always inline the function - if such a thing existed.